.github/workflows/publish_beta.yaml

```diff
@@ -0,0 +1,47 @@
+name: Publish ROCK (beta)
+on:
+  workflow_dispatch:
```

Comment:
this means it can only be manually triggered, is that correct?

Reply:
yes, [just FYI workflow_dispatch]( https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch)